tests/acceptance/features/apiOcisSpecific/apiWebdavProperties2-getFileProperties.feature

```diff
@@ -49,62 +19,6 @@ Feature: get file properties
-    And user "Alice" has uploaded file with content "uploaded content" to "<folder_name>/file2.txt"
-    When user "Alice" gets the properties of folder "<folder_name>" with depth 1 using the WebDAV API
-    Then the value of the item "//d:response[1]/d:href" in the response to user "Alice" should match "/remote\.php\/<expected_href>\//"
-    And the value of the item "//d:response[2]/d:href" in the response to user "Alice" should match "/remote\.php\/<expected_href>\/file1.txt/"
-    And the value of the item "//d:response[3]/d:href" in the response to user "Alice" should match "/remote\.php\/<expected_href>\/file2.txt/"
-    Examples:
-      | dav_version | folder_name     | expected_href                                                             |
-      | new         | /upload         | dav\/files\/%username%\/upload                                                 |
-      | new         | /strängé folder | dav\/files\/%username%\/str%C3%A4ng%C3%A9%20folder                             |
-      | new         | /C++ folder     | dav\/files\/%username%\/C\+\+%20folder                                           |
-      | new         | /नेपाली         | dav\/files\/%username%\/%E0%A4%A8%E0%A5%87%E0%A4%AA%E0%A4%BE%E0%A4%B2%E0%A5%80 |
-      | new         | /folder #2.txt  | dav\/files\/%username%\/folder%20%232\.txt                                     |
-      | new         | /folder &2.txt  | dav\/files\/%username%\/folder%20&2\.txt                                       |
-
-  @issue-ocis-reva-214 @skipOnOcis-EOS-Storage @issue-ocis-reva-265 @skipOnOcis-OCIS-Storage
-  # after fixing all issues delete this Scenario and use the one from oC10 core
-  Scenario Outline: Do a PROPFIND of various folder names
-    Given using <dav_version> DAV path
-    And user "Alice" has created folder "<folder_name>"
-    And user "Alice" has uploaded file with content "uploaded content" to "<folder_name>/file1.txt"
-    And user "Alice" has uploaded file with content "uploaded content" to "<folder_name>/file2.txt"
-    When user "Alice" gets the properties of folder "<folder_name>" with depth 1 using the WebDAV API
-    Then the value of the item "//d:response[1]/d:href" in the response to user "Alice" should match "/remote\.php\/<expected_href>\//"
-    And the value of the item "//d:response[2]/d:href" in the response to user "Alice" should match "/remote\.php\/<expected_href>\/file1.txt/"
-    And the value of the item "//d:response[3]/d:href" in the response to user "Alice" should match "/remote\.php\/<expected_href>\/file2.txt/"
-    Examples:
-      | dav_version | folder_name     | expected_href                                                             |
-      | old         | /folder ?2.txt  | webdav\/folder%20%3F2\.txt                                                |
-      | new         | /folder ?2.txt  | dav\/files\/%username%\/folder%20%3F2\.txt                                     |
```

Comment:
Looks good to delete these "local" test scenarios. Various core scenarios are now passing, which is great.
Requiring exact values like `%3F` is problematic in tests like this - both `%3F` and `%3f` are valid. I raised issue https://github.com/owncloud/core/issues/38367 to get the core API tests sorted out so that they have the needed flexibility. That way, in future, we can have implementations that return either case and the test suite will allow it.